x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/util/Check.java

```diff
@@ -36,4 +36,14 @@ public static void notNull(Object object, String message, Object... values) {
             throw new SqlIllegalArgumentException(message, values);
         }
     }
+    
+    public static void isNumberOutOfRange(Object object, String objectName, Long from, Long to) {
+        if (!(object instanceof Number)) {
+            throw new SqlIllegalArgumentException("A number is required; received [{}]", object);
```

Comment:
Since the object name is available, we might as well use it? We also want to log object's class name, not value.
```suggestion
            throw new SqlIllegalArgumentException("A number is required for[{}]; received [{}]", objectName, object.getClass());
```

Reply:
Thanks for the advice.